sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Azure Service Bus is a fully managed enterprise integration message broker. Service Bus can decouple applications and services. Service Bus offers a reliable and secure platform for asynchronous transfer of data and state. This client library allows for both sending and receiving messages using Azure Service Bus. For more information about Service Bus, see https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview</Description>
-    <Version>7.1.0-beta.1</Version>
-    <ApiCompatVersion>7.0.0</ApiCompatVersion>
+    <Version>7.0.1</Version>
+    <ApiCompatVersion>7.0.1</ApiCompatVersion>
```

Comment:
Is this supposed to be set manually?

Reply:
/cc @weshaggard 
There are no API changes in this release, but I bumped it up anyway.

Reply:
Yes.  Once you release, an auto-gen PR will increment it to the next beta.

Reply:
You don't need to ever change this manually especially to the version that's not published yet. The build tries to compare against it and would fail.